api-private.swagger.yaml

```diff
@@ -358,6 +353,7 @@ definitions:
     required:
       - job_uuid
       - user_uuid
+      - latest
```

Comment:
The `latest` parameter is not required, it should default to false like it's already implemented in Node.

Reply:
Thanks; fixed